features/step-definitions/clj-refactor-steps.el

```diff
@@ -66,20 +66,24 @@
     (run-rename-symbol "example.two" '(6  7  1  10  "star*"  "tmp/src/example/two.clj"  "" 8  8  17  27  "star*"  "tmp/src/example/one.clj"  "") "asterisk*")))
 
 (Given "^I call the add-missing-libspec callback directly with mock data to import"
-  (lambda ()
-    (cljr--add-missing-libspec "Date" "java.util.Date" "import")))
+       (lambda ()
+         (cljr--add-missing-libspec "Date" '((java.util.Date :class)))))
 
 (Given "^I call the add-missing-libspec callback directly with mock data to refer split"
-  (lambda ()
-    (cljr--add-missing-libspec "split" "clojure.string" "require")))
+       (lambda ()
```

Comment:
No, I just copied those from the docs back in 2012 and never reindented the
buffer, it seems.

On Thu, Mar 5, 2015 at 7:22 PM Lars Andersen <notifications@github.com>
wrote:

> In features/step-definitions/clj-refactor-steps.el
> <https://github.com/clojure-emacs/clj-refactor.el/pull/114#discussion_r25887562>
> :
>
> >
> >  (Given "^I call the add-missing-libspec callback directly with mock data to refer split"
> > -  (lambda ()
> > -    (cljr--add-missing-libspec "split" "clojure.string" "require")))
> > +       (lambda ()
>
> Did you add indentation rules manually to get Given to indent in this
> manner, @magnars <https://github.com/magnars>?
>
> aggressive-indent-mode ruined this for me and I was too lazy to turn it
> off to indent manually :p
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clojure-emacs/clj-refactor.el/pull/114/files#r25887562>
> .
>